static/js/containers/LearnerSearchPage.js

```diff
@@ -0,0 +1,54 @@
+// @flow
```

Comment:
noticed that we have `LearnerSearchPage` and `LearnersSearch` components. 

1. we should rename the latter to `LearnerSearch`
1. the naming is a little confusing. 
1. do we have a compelling reason to have two different components for these (apart from not having a huge-ass js file)? i know we have a pattern of using container files. i guess i'm not sure what we should be considering a container, or if it's worth it to have a container that will only be used for one page